lib/routes/charge/items/item_avatar.dart

```diff
@@ -91,17 +138,67 @@ class _FileImageAvatar extends StatelessWidget {
   }
 }
 
-class _NetworkImageAvatar extends StatelessWidget {
+class _UnknownAvatar extends StatelessWidget {
   final double radius;
-  final String avatarURL;
+  final String itemName;
+  final bool useDecoration;
 
-  _NetworkImageAvatar(this.avatarURL, this.radius);
+  _UnknownAvatar(this.radius, this.itemName, this.useDecoration);
 
   @override
   Widget build(BuildContext context) {
-    return CircleAvatar(
-      radius: radius,
-      backgroundImage: AdvancedNetworkImage(avatarURL, useDiskCache: true),
+    return Container(
+      width: radius * 2,
+      height: radius * 2,
+      decoration: useDecoration
+          ? BoxDecoration(
+              shape: BoxShape.circle,
+              border: Border.all(
+                  color: Colors.white, width: 1.0, style: BorderStyle.solid),
+              image: DecorationImage(
+                  colorFilter: ColorFilter.mode(
+                      Theme.of(context).primaryColorLight, BlendMode.srcATop),
+                  image: AssetImage("src/images/avatarbg.png"),
+                  fit: BoxFit.cover))
+          : null,
+      child: (itemName != null && itemName.isNotEmpty)
+          ? Center(
+              child: Text(
+              _getFirstTwoLetters(),
+              style: TextStyle(
+                  fontSize: useDecoration ? 48 : radius,
+                  color: Color.fromRGBO(255, 255, 255, 0.88),
+                  decoration: TextDecoration.underline,
+                  letterSpacing: 0.0,
+                  fontFamily: "IBMPlexSans"),
+            ))
+          : Column(
```

Comment:
It doesn't look like the "Select Image" case belongs here. Instead at a higher level we should decide if to show an avatar (in case we have one) or to render a "Select Image" widget.

Reply:
[Done](https://github.com/breez/breezmobile/pull/319/commits/cefa0a16444237d5af2f2afbf11291f438b2353b)